packages/contracts/contracts/L1/rollup/CanonicalTransactionChain.sol

```diff
@@ -48,6 +48,14 @@ contract CanonicalTransactionChain is ICanonicalTransactionChain, Lib_AddressRes
     uint256 public maxTransactionGasLimit;
 
 
+    /***************
+     * Queue State *
+     ***************/
+
+    uint40 private nextQueueIndex; // index of the first queue element not yet included
```

Comment:
made this private to avoid modifying the interface, and force using the `getNextQueueIndex()` function.